content/en/infrastructure/containers/container_images.md

```diff
@@ -76,6 +70,81 @@ Or, add the following to your `values.yaml` Helm configuration file:
 datadog:
   containerImageCollection:
     enabled: true
+```
+[1]: https://github.com/DataDog/helm-charts/blob/main/charts/datadog/values.yaml#L651
+{{% /tab %}}
+
+{{% tab "ECS EC2" %}}
+
+To enable container image collection on your [ECS EC2 instances][1], add the following environment variables to your `datadog-agent` container definition:
+
+```yaml
+{
+    "containerDefinitions": [
+        {
+            "name": "datadog-agent",
+             ...
+            "environment": [
+              ...
+              {
+                "name": "DD_CONTAINER_IMAGE_ENABLED",
+                "value": "true"
+              }
+            ]
+        }
+    ]
+  ...
+}
+```
+
+[1]: https://docs.datadoghq.com/containers/amazon_ecs/?tab=awscli#setup
+
+{{% /tab %}}
+
+{{% tab "Hosts" %}}
+
+Add the following to your `datadog.yaml` configuration file:
+
+```yaml
+container_image:
+  enabled: true
+```
+
+[1]: /containers/amazon_ecs/?tab=awscli#setup
+{{% /tab %}}
+{{< /tabs >}}
+
+#### Enable SBOM collection
+
+The following instructions enable the [Software Bill of Materials][5] (SBOM) collection in the Datadog Agent for CSM Vulnerabilities. This allows you to scan the libraries in container images to detect vulnerabilities. Vulnerabilities are evaluated and scanned against your containers every hour. Vulnerability management for container images is included in [CSM Pro and Enterprise plans][10].
```

Comment:
It can be misleading with "This allows you.. ", it should probably imply automatic scanning, suggestion below, let's confirm with folks from Documentation and @rajat-luthra-ddog.
```suggestion
The following instructions enable the [Software Bill of Materials][5] (SBOM) collection in the Datadog Agent for CSM Vulnerabilities. SBOM collection enables automatic detection of container image vulnerabilities. Vulnerabilities are evaluated and scanned against your containers every hour. Vulnerability management for container images is included in [CSM Pro and Enterprise plans][10].
```

Reply:
I agree with your suggestion. I've included it in https://github.com/DataDog/documentation/pull/22970/commits/77eabfd9f2bbd12679b2a708b28d379cfd6a380d.